### PR TITLE
5663 Search Results Content and Misc Bugs

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#5663

*Description of changes:*

* Updated search results content.

* Added condition to identify issues in requests and render the appropriate message.

* disabled QUILL input field when the value has met approval conditions.

* Bug fix on Search Input overriding input rules.

* Minor styling adjustments.
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/new-request/analyze-results.vue [hunk 1]
```diff
@@ -490,23 +489,31 @@ export default class AnalyzeResults extends Vue {
       return {
         class: 'approved',
         icon: 'mdi-check-circle',
-        text: 'Name Available',
+        text: 'Name Ready for Review',
         showNextLines: true
       }
     }
     let { length } = this.json.issues
+    if (length >= 1) {
```
Comment (lekshmimallika-aot) [inline client/src/components/new-request/analyze-results.vue:497]: Is this true for all issue types. Are there any issue types for which we display just a message to the user and still show the proceed option

**Review by vysakh-menon-aot — APPROVED** [review]

**Review by lekshmimallika-aot — APPROVED** [review]
Looks great. Just one question that may have to be clarified with BA

File: client/src/components/new-request/analyze-results.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/analyze-results.vue:497]: I'm really not sure 😬  
I'll review the designs i have, but every issue seems to require some sort of action, in my limited* experience.

File: client/src/views/landing.vue [hunk 2]
```diff
@@ -13,8 +13,9 @@
               </template>
               <p>The Name Request website is available as a Beta version. Name Requests obtained through the Beta are
                 official name requests and can be used in the province of British Columbia.</p>
-              <p>As part of a new <u>agile software development process</u>, the Name Request website will be
-                continually updated and improved based on feedback from citizens and businesses in BC.</p>
+              <p>As part of a new <a class="" :href="agileUrl">agile software development process</a>, the Name Request
```
Comment (severinbeauvais) [inline client/src/views/landing.vue:16]: What's the URL?
Should "agile" be "Agile"?

File: client/src/views/landing.vue [hunk 3]
```diff
@@ -105,5 +108,11 @@ export default class Landing extends Vue {
   font-size: 19px;
   font-weight: bold;
   text-transform: uppercase;
+  &:hover {
+    cursor: default;
+  }
+}
+.v-application a {
+  color: white;
```
Comment (severinbeauvais) [inline client/src/views/landing.vue:116]: This seems overly broad. Can you specify this more?

**Review by severinbeauvais — APPROVED** [review]
Couple of questions but otherwise good to go.

File: client/src/views/landing.vue (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/views/landing.vue:16]: Oops will do!
Design shows it as lower case. Should it be capitalized?

File: client/src/views/landing.vue (hunk 3 above)
Comment (cameron-freshworks) [reply client/src/views/landing.vue:116]: Absolutely 👍

File: client/src/views/landing.vue (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/views/landing.vue:16]: Fixed ( using vuetify color class helpers... ) I keep forgetting about these. Like i said, changed my life lol

File: client/src/views/landing.vue (hunk 3 above)
Comment (cameron-freshworks) [reply client/src/views/landing.vue:116]: Removed!

File: client/src/components/new-request/analyze-results.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/analyze-results.vue:497]: There is some more flow changes with regards to names that are not analyzed, which we discovered as a result of this. But there is no action required here, as it is it's own can of worms to address in a new ticket 😁

**Pull request merged**